### DISCUSSION TITLE
how to change background of chatmessage

### DISCUSSION THREAD

**Discussion by 1945083536** [zauberzeug/nicegui#3248] [Q&A] [answered]
### Question

I want to let the background of chatmessage is red. below is my code.
![image](https://github.com/zauberzeug/nicegui/assets/13993532/f67589d0-a198-4770-b041-5635c99de97b)

but the result is shown below:
![image](https://github.com/zauberzeug/nicegui/assets/13993532/1365d7bb-ef4f-45de-b8d6-9fb3adc1ba4c)

as you can see, there are some places still green.

How to change entire background color? Thanks

**Comment by python-and-novella** [comment, marked as answer]
try this
```py
message = ui.chat_message(name='Robot').props('bg-color="red"')
with message:
    ui.label('test')
```
check https://quasar.dev/vue-components/chat/#customization
you will know how to customize, Properties in vue-components, do same in nicegui's control within props function.

**Reply by 1945083536** [reply]
Thanks! It works.
